client/my-sites/theme/controller.jsx

```diff
@@ -77,7 +77,10 @@ export function fetchThemeDetailsData( context, next ) {
 export function details( context, next ) {
 	const { slug } = context.params;
 	const user = getCurrentUser( context.store.getState() );
-	const themeName = ( getThemeDetails( context.store.getState(), slug ) || false ).name;
+	const themeDetails = getThemeDetails( context.store.getState(), slug ) || false;
+	const themeName = themeDetails.name;
+	const themeDescription = themeDetails.description;
+	const themeCanonicalUrl = `https://wordpress.com/theme/${ slug }`;
```

Comment:
In the long run, I'd love us to use `getDetailsUrl()` here (which isn't possible now as it expects an entire theme object, not just a slug). Could you add a `// TODO: ...` comment to that effect find this instance easier once I've tweaked `getDetailsUrl()`?

Also, I think it's fine to directly set the `props`' object `description` and `canonicalUrl` fields, without the intermediary constants.